api/queries_org.go

```diff
@@ -42,45 +42,6 @@ func OrganizationProjects(client *Client, repo ghrepo.Interface) ([]RepoProject,
 	return projects, nil
 }
 
-// OrganizationProjectsV2 fetches all open projectsV2 for an organization.
-func OrganizationProjectsV2(client *Client, repo ghrepo.Interface) ([]RepoProjectV2, error) {
```

Comment:
This just got moved not actually deleted. Consolidated it in `queries_projects_v2.go`.